apps/condo/lang/en/en.json

```diff
@@ -1577,5 +1577,65 @@
+  "banking.integration.SBBOL.name": "SBBOL",
+  "banking.integration.1CClientBankExchange.name": "1C",
+  "banking.category.maintenance_and_current_repair_of_common_property_in_an_apartment_building.name": "Maintenance and current repair of common property in an apartment building",
+  "banking.category.apartment_building_management.name": "Apartment building management",
+  "banking.category.overhaul.name": "Overhaul",
+  "banking.category.drainage.name": "Drainage",
+  "banking.category.water_supply.name": "Water supply",
+  "banking.category.garbage_collection_and_disposal.name": "Garbage collection and disposal",
+  "banking.category.gas.name": "Gas",
+  "banking.category.hot_water_supply.name": "Hot water supply",
+  "banking.category.additional_services.name": "Additional services",
+  "banking.category.thermal_energy.name": "Thermal energy",
+  "banking.category.cold_water_supply.name": "Cold water supply",
+  "banking.category.electric_energy.name": "Electric energy",
+  "banking.costItem.emergency_service.name": "Emergency service",
+  "banking.costItem.rent.name": "Rent",
+  "banking.costItem.banking_services.name": "Banking services",
+  "banking.costItem.maintenance_and_operation.name": "Maintenance and operation",
+  "banking.costItem.overhaul.name": "Overhaul",
+  "banking.costItem.water_supply.name": "Water supply",
+  "banking.costItem.compensation_for_damage.name": "Compensation for damage",
+  "banking.costItem.garbage_collection_and_disposal.name": "Garbage collection and disposal",
+  "banking.costItem.gas.name": "Gas",
+  "banking.costItem.deratization_and_disinfestation.name": "Deratization and disinfestation",
+  "banking.costItem.materials_for_repair_and_maintenance.name": "Materials for repair and maintenance",
+  "banking.costItem.purchase_of_machinery_and_equipment.name": "Purchase of machinery and equipment",
+  "banking.costItem.employee_wages.name": "Employee wages",
+  "banking.costItem.rental_fee.name": "Rental fee",
+  "banking.costItem.concierges.name": "Concierges",
+  "banking.costItem.materials_for_maintenance_and_current_repair.name": "Materials for maintenance and current repair",
+  "banking.costItem.taxes.name": "Taxes",
+  "banking.costItem.equipment.name": "Equipment",
+  "banking.costItem.maintenance_of_access_control_systems.name": "Maintenance of access control systems",
+  "banking.costItem.intercom_service.name": "Intercom service",
+  "banking.costItem.maintenance_and_inspection_of_metering_devices.name": "Maintenance and inspection of metering devices",
+  "banking.costItem.maintenance_of_elevator_equipment.name": "Maintenance of elevator equipment",
+  "banking.costItem.maintenance_of_fireplaces_and_hearths.name": "Maintenance of fireplaces and hearths",
+  "banking.costItem.maintenance_of_ventilation_systems.name": "Maintenance of ventilation systems",
+  "banking.costItem.maintenance_of_video_surveillance_systems.name": "Maintenance of video surveillance systems",
```

Comment:
Please consider following difference between CCTV and surveillance systems

Key difference: Basically, CCTV and Surveillance Camera are video cameras working on advanced technologies. CCTV known as Closed-circuit television transmits a signal to a specific monitor located at a distant place; while Surveillance Camera often works on IP networks which connect the security (surveillance) cameras from the remote location to the main location

Is it sure a system of cameras, transmitting signals over IP networks?

Reply:
I also can't be sure if security systems use a closed data loop, so I called it by a more general name

Reply:
ok